net/stunnel/files/stunnel.init

```diff
@@ -81,7 +81,10 @@ print_list_colon() {
 
 service_section() {
 	local cfg="$1"
-	local accept_host accept_port
+	local accept_host accept_port enabled
+
+	config_get enabled "$cfg" enabled 1
+	[ "$enabled" -gt 0 ] && return 0
```

Comment:
@feckert 
Just wondering if this is right.
Is that really "if enabled>0 then return immediately without processing the config".
Or do I get it wrong?

Reply:
@hnyman thannk you very much for your hint
it was an inattention from my side :-(
I think a better solution is

> config_get_bool enabled "${cfg}" 'enabled' '1'
[ ${enabled} -gt 0 ] || return 0